examples/evaluation/Getting_Started_with_OpenAI_Evals.ipynb

```diff
@@ -391,6 +396,8 @@
    "source": [
     "These CLIs can accept various flags to modify their default behavior. You can run `oaieval --help` to see a full list of CLI options. \n",
     "\n",
+    "In the statement below, `oaieval` will look for `spider-sql` eval YAML file under the directory `evals/registry/evals` in a format as specified in cell number 4 above. The path to the eval dataset is provided in the eval YAML file under `args:` parameter. In this case the path is specified as `samples_jsonl: sql/spider_sql.jsonl` with the file content in jsonl format (as generated in step 3 above)\n",
```

Comment:
suggestion: oaieval will search for the spider-sql eval YAML file in the evals/registry/evals directory, following the format specified in cell 4 above. The path to the eval dataset is specified in the eval YAML file under the args: parameter as samples_jsonl: sql/spider_sql.jsonl, with the file content in JSONL format (as generated in step 3 above).